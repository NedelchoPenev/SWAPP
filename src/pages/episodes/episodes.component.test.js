import React from 'react';
import { shallow } from 'enzyme';

import EpisodesPage from './episodes.component';

describe('EpisodesPage component', () => {

  it('should render EpisodesPage component', () => {
    const mockProps = {
      allEpisodes: {
        edges: [{
          node: {
            episodeId: 'film.2'
          }
        },
        {
          node: {
            episodeId: 'film.1'
          }
        }]
      }
    }

    expect(shallow(<EpisodesPage {...mockProps}/>)).toMatchSnapshot();
  });
});