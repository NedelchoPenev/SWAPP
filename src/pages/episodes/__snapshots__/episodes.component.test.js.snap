// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EpisodesPage component should render EpisodesPage component 1`] = `
<styled.div>
  <withRouter(EpisodeCard)
    epsisode={
      Object {
        "episodeId": "film.2",
      }
    }
    key="film.2"
  />
  <withRouter(EpisodeCard)
    epsisode={
      Object {
        "episodeId": "film.1",
      }
    }
    key="film.1"
  />
</styled.div>
`;
